### DISCUSSION TITLE
Installation of podman not working as expected

### DISCUSSION THREAD

**Discussion by soupdiver** [Homebrew/discussions#3091] [Linux] [answered]
On my Ubuntu 20.04 system I installed podman via `brew install podman`
However this is giving me a not satisfying result.

1) After installation there is no `podman` binary. Only `podman-remote`
  I guess this is why: https://github.com/Homebrew/homebrew-core/blob/master/Formula/podman.rb#L53-L59
  But why will it only install `podman-remote` on non-mac?
2) Trying any command through `podman-remote` gives me an error like `Cannot connect to Podman. Please verify your connection to the Linux system using podman system connection list, or try podman machine init and podman machine start to manage a new Linux VM...`
Why does it require a VM? When installing `podman` through apt this all just works?

What is the difference here? 🤔

**Comment by gromgit** [comment, marked as answer]
I don't use `podman`, but it looks like https://github.com/Homebrew/homebrew-core/pull/83578 has the answers you seek.

**Comment by nozzlegear** [comment]
I'm actually having this problem as well. Podman is no longer publishing to the kubic PPA on Ubuntu, and the official Ubuntu versions are still on 3.x. I need to get Podman 4.x, so I figured I'd just install Homebrew and get it that way. Unfortunately it doesn't work since there's no `podman` binary included. It looks like my only choice is to build it from source now.

**Reply by SMillerDev** [reply]
Yeah unfortunately podman hardcode a lot of paths that make it hard to package in Homebrew.

**Reply by nozzlegear** [reply]
Ah I see, that's unfortunate. I did manage to get podman built/installed from source on Linux at least, and continuing to use it via homebrew on my mac.
